dev/tools/controllerbuilder/pkg/commands/exportcsv/prompt.go

```diff
@@ -44,6 +44,16 @@ type PromptOptions struct {
 	StrictInputColumnKeys []string
 }
 
+func (o *PromptOptions) InitDefaults() error {
```

Comment:
Suggest expose this defaulting, like in cmd flag, make rules, or bash script. A common pain point we have is that some variables are modified all over the place especially in the release pipeline. Say if we later on change the proto to somehwere like `pkg/internal/third_party/googleapis/1.123/` (for versioning), it wold be hard to discover this function.